blocks/clickable-columns/clickable-columns.css

```diff
@@ -0,0 +1,49 @@
+.clickable-columns {
+  margin: 16px 0;
+}
+
+.clickable-columns > div {
+  display: flex;
+  flex-direction: column;
+}
+
+.clickable-columns img {
+  width: 100%;
+}
+
+.clickable-columns .icon {
+  width: auto !important;
+}
+
+.clickable-columns > div > div {
+  padding: 24px;
+  order: 1;
+}
+
+.clickable-columns > div > .columns-img-col {
+  order: 0;
+}
+
+.clickable-columns > div > .columns-img-col img {
+  display: block;
+}
+
+@media (width >= 900px) {
```

Comment:
@kronnox there may be a browser compat issue with this style of media query as we found on a different project. Please see @seanathero's slack comment for more detail: https://herodigital.slack.com/archives/C06JE9U7Y74/p1717612144522739